pkg/plugin/connector/standalone/registry.go

```diff
@@ -166,8 +164,9 @@ func (r *Registry) loadSpecifications(pluginPath string) (pconnector.Specificati
 	dispenser, err := NewDispenser(
 		zerolog.Nop(),
 		pluginPath,
+		// todo are the grpc target and token really needed?
```

Comment:
We can keep them in case there's some code that expects a value there, but most probably the connector won't need them here. The only use case I see its the grpc target, if the connector wants to check that a service is available or something.